### PR TITLE
LL-5232 USB troubleshooting wizard for LLD

### PR DESCRIPTION
**Pull request by juan-cortes**
## 🦒 Context (issues, jira)

> ~Leaving as draft until I've finished the spectron testing~ Now with spectron coverage
[LL-5232]
## 💻  Description / Demo (image or video)
image

This pr introduces a new wizard for troubleshooting the most common USB problems from our users, it's a pretty straightforward trial and error approach, with a continuous connection tested component to see if we've succeeded or not. If in the end we've not succeeded we can either trigger the Device repair modal for LNS or forward the case to support for other devices.

It can be accessed whenever there's a device action and we don't have a device plugged in, after 15 seconds there should be a new UI element that allows us to see this.

QA test plan notes:
- If we restart the app while it's showing a solution, the app should automatically show that solution on relaunch.
- Navigating away from the troubleshooter is the same as exiting it, meaning a restart will no longer bring us back there.
- A successful connection can be achieved by connecting a device an unlocking it in any of the solutions                                  
- There's an exposed .env to simulate windows/mac/linux `USBTROUBLESHOOTING_PLATFORM=mac`

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [x] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

[LL-5232]: https://ledgerhq.atlassian.net/browse/LL-5232

### CODE REVIEW

**Comment by juan-cortes** [comment]
Redoing all the wording because it was already present

File: src/renderer/ModalsLayer.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ const ModalsLayer = ({ visibleModals }: *) => {
     ({ name, MODAL_SHOW_ONCE }) =>
       MODAL_SHOW_ONCE && global.sessionStorage.setItem(name, Date.now()),
   );
+  useEffect(() => console.log({ visibleModals, filteredModals }), [filteredModals, visibleModals]);
```
Comment (LFBarreto) [inline src/renderer/ModalsLayer.js:39]: do you need a console log here?

File: src/renderer/screens/USBTroubleshooting/solutions/RepairFunnel.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+import nanoS from "~/renderer/components/Onboarding/Screens/SelectDevice/assets/nanoS.svg";
+import nanoBlue from "~/renderer/components/Onboarding/Screens/SelectDevice/assets/nanoBlue.svg";
+import { registerAssets } from "~/renderer/components/Onboarding/preloadAssets";
+registerAssets([nanoX, nanoS, nanoBlue]);
+
+const DeviceSelectContainer: ThemedComponent<*> = styled.div`
+  display: flex;
+  flex-direction: row;
+
+  & > * {
```
Comment (LFBarreto) [inline src/renderer/screens/USBTroubleshooting/solutions/RepairFunnel.js:26]: could you not? :vomiting_face: 
but if you cant just keep that :laughing:

**Review by LFBarreto — DISMISSED** [review]

**Review by LFBarreto — APPROVED** [review]

**Comment by juan-cortes** [comment]
cc @machard 39b3dfe0d7a38e49173c2864780b5d7b286248fd  includes that change you gave me to ensure container rebuild

**Review by ychen-ledger — DISMISSED** [review]
Validated with(all rest improvements will be done in the epic https://ledgerhq.atlassian.net/browse/LL-7425): 

- Mac/Linux/Windows
- LLD 2.33.1
- Automated update udev is abandoned for linux

**Pull request merged**